js-src/compat/themes/blocksy/color-mode-switch.js

```diff
@@ -0,0 +1,144 @@
+/**
+ * Sync FC color mode with theme's
+ *
+ * DEPENDS ON:
+ * - jQuery // Interact with WooCommerce events
+ */
+(function (root, factory) {
+	if ( typeof define === 'function' && define.amd ) {
+		define([], factory(root));
+	} else if ( typeof exports === 'object' ) {
+		module.exports = factory(root);
+	} else {
+		root.BlocksyColorModeSwitch = factory(root);
+	}
+})(typeof global !== 'undefined' ? global : this.window || this.global, function (root) {
+
+	'use strict';
+
+	var $ = jQuery;
+	var _hasJQuery = ( $ != null );
+
+	var _hasInitialized = false;
+	var _publicMethods = {};
+	var _settings = {
+		switchColorModeNonce: '', // Value updated during runtime
+	};
+
+
+
+	/**
+	 * METHODS
+	 */
+
+
+
+	/**
+	 * Get cookie value by name.
+	 * 
+	 *  @param  {String}  cname  Name of the cookie.
+	 */
+	var getCookie = function( cname ) {
+		// Append "=" to the cookie name
+		let name = cname + "=";
```

Comment:
Our code base uses ES5 syntax, so `let` and `const` is not to be used as it comes from ES6 syntax.